beater/pub.go

```diff
@@ -85,14 +90,22 @@ func (p *publisher) Stop() {
 // Send tries to forward pendingReq to the publishers worker. If the queue is full,
 // an error is returned.
 // Calling send after Stop will return an error.
-func (p *publisher) Send(req pendingReq) error {
+func (p *publisher) Send(ctx context.Context, req pendingReq) error {
 	p.m.RLock()
 	defer p.m.RUnlock()
 	if p.stopped {
 		return errChannelClosed
 	}
 
+	span, ctx := elasticapm.StartSpan(ctx, "Send", "Publisher")
```

Comment:
what is the difference between `elasticapm.StartSpan` and `tx.StartSpan` and why you use one over the other one?

Reply:
elasticapm.StartSpan pulls info (transaction and parent span) out of a context.Context, tx.StartSpan does not.